GA-worksheet.md

```diff
@@ -26,44 +26,46 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
-2. What is normalize.css?
+The advantages of <link> tag is it includes CSS from an external file instead of writting it all out in HTML file.
+2. What is normalize.css? Normalized.css is according to its websites and makes browser render all elements more consistently in line with modern standards so the page will look the same in any browser.
 
 3. Why do we need to add a padding property to the `<ul>` tag?
-
-4. What does the “inline” property do to a list of items?
+We use padding property to the <ul> to make the list in the center.
```

Comment:
In the dash tutorial, the `text-align: center' in the header is actually what makes it centered.
There is default padding on the left for unordered lists to accommodate the bullet points, so adding padding to all sides makes this particular ul centered. Without it, it would stay shifted to the right.